src/main/java/com/northconcepts/datapipeline/foundations/examples/pipeline/InferDateTimeFromCSVFile.java

```diff
@@ -0,0 +1,47 @@
+package com.northconcepts.datapipeline.foundations.examples.pipeline;
+
+import com.northconcepts.datapipeline.foundations.file.LocalFileSource;
+import com.northconcepts.datapipeline.foundations.pipeline.Pipeline;
+import com.northconcepts.datapipeline.foundations.pipeline.dataset.Column;
+import com.northconcepts.datapipeline.foundations.pipeline.dataset.Dataset;
+import com.northconcepts.datapipeline.foundations.pipeline.dataset.MemoryDataset;
+import com.northconcepts.datapipeline.foundations.pipeline.input.CsvPipelineInput;
+import com.northconcepts.datapipeline.foundations.time.DateTimePattern;
+
+import java.util.Map;
+import java.util.concurrent.atomic.LongAdder;
+
+public class InferDateTimeFromCSVFile {
+    public static void main(String[] args) {
+        CsvPipelineInput pipelineInput = new CsvPipelineInput()
+                .setFileSource(new LocalFileSource().setPath("example/data/input/DateTimeData.csv"))
+                .setFieldNamesInFirstRow(true);
+
+        Pipeline pipeline = new Pipeline();
+        pipeline.setInput(pipelineInput);
+
+        Dataset dataset = new MemoryDataset(pipeline);
+        dataset.load().waitForColumnStatsToLoad();
+        dataset.setCollectUniqueValues(true);
```

Comment:
`setCollectUniqueValues(true)` should be called before `load()`, but do we really need it set to true?

Reply:
I removed this line